### PR TITLE
loginctl: add page

### LINKED ISSUE #9005: page request: loginctl
### Command description

Controls the `systemd` login manager - https://manned.org/loginctl.1

### Command details

_No response_

### OS

- [ ] Android
- [X] Linux
- [ ] macOS
- [ ] sunOS
- [ ] Windows

### PR DESCRIPTION
**Pull request by Nikhil-Samant**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

closes #9005

### CODE REVIEW

File: pages/linux/loginctl.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# loginctl
+
+> Manage the systemd login manager.
+> More information: <https://manned.org/loginctl>.
+
+- Print the current sessions:
```
Comment (marchersimon) [inline pages/linux/loginctl.md:6]: It's a bit odd to use "the" with a plural noun. How about
```suggestion
- Print all current sessions:
```

File: pages/linux/loginctl.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+`loginctl list-sessions`
+
+- Print all the properties of a specific session:
```
Comment (marchersimon) [inline pages/linux/loginctl.md:10]:
```suggestion
- Print all properties of a specific session:
```

File: pages/linux/loginctl.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+`loginctl show-session {{sessionId}} --all`
```
Comment (marchersimon) [inline pages/linux/loginctl.md:12]: Argument placeholders should be snake_case. Please also change this for the other occurences.
```suggestion
`loginctl show-session {{session_id}} --all`
```

File: pages/linux/loginctl.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+- Print the user details:
```
Comment (marchersimon) [inline pages/linux/loginctl.md:14]:
```suggestion
- Print the details of a specific user:
```

File: pages/linux/loginctl.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+`loginctl show-user {{userName}}`
+
+- Print specific user property:
```
Comment (marchersimon) [inline pages/linux/loginctl.md:18]:
```suggestion
- Print a property of a specific user:
```

File: pages/linux/loginctl.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+`loginctl show-user {{userName}} --property={{propertyName}}`
+
+- Execute `loginctl` operation remotely:
```
Comment (marchersimon) [inline pages/linux/loginctl.md:22]: Is this still correct?
```suggestion
- Execute a `loginctl` operation on a remote host:
```

File: pages/linux/loginctl.md (hunk 3 above)
Comment (Nikhil-Samant) [reply pages/linux/loginctl.md]: made the changes for all other occurrences

**Review by adrienthebo — APPROVED** [review]
LGTM!

**Review by navarroaxel — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**